templates/noKibana/accounts.go

```diff
@@ -1,12 +1,13 @@
 package noKibana
 
 import (
-	"github.com/ElrondNetwork/elastic-indexer-go/templatesConfig"
+	"github.com/ElrondNetwork/elastic-indexer-go/templates"
 )
 
-type Array = templatesConfig.Array
-type Object = templatesConfig.Object
+type Array = templates.Array
```

Comment:
Do we need to redefine the same Array and Object? Why? If this is required, please add comments.

Reply:
I like these renames because I no longer have to use the package name from where is Array and Object type  in this package `noKibana`